cypress.config.mjs

```diff
@@ -1,48 +1,58 @@
 import { defineConfig } from 'cypress';
-import webpackProcessor from '@cypress/webpack-preprocessor';
+import vitePreprocessor from 'cypress-vite';
+import { cypressConfig } from './vite.config.js';
 
 import coveragePlugin from './test/plugins/coverage.js';
 import fakerPlugin from './test/plugins/faker-generator.js';
-import webpackOptions from './test/webpack.config.js';
 
 fakerPlugin();
 
+process.env.NODE_ENV = 'test';
+
 function setupNodeEvents(on, config) {
   coveragePlugin(on, config);
-  on('file:preprocessor', webpackProcessor({ webpackOptions }));
+  on(
+    'file:preprocessor',
+    vitePreprocessor({
+      configFile: false,
```

Comment:
loading the base configFile causes issues as we do not want to copy the public assets into the cypress cache